pkg/network/macvtap.go

```diff
@@ -58,3 +60,15 @@ func fillMacvtapDefaults(conf *cnao.NetworkAddonsConfigSpec, previousConf *cnao.
 func hasDevicePluginConfigMapNameDefined(conf *cnao.NetworkAddonsConfigSpec) bool {
 	return conf != nil && conf.MacvtapCni != nil && conf.MacvtapCni.DevicePluginConfig != ""
 }
+
+func validateMacvtap(conf *cnao.NetworkAddonsConfigSpec, openshiftNetworkConfig *openshiftoperatorv1.Network) []error {
+	if conf.MacvtapCni == nil {
+		return nil
+	}
+
+	if openshiftNetworkConfig != nil {
+		return []error{fmt.Errorf("`macvtap` configuration is not supported on Openshift yet")}
```

Comment:
While "yet" often suggests the possibility of change or an expectation of something happening in the future, it can also be used to emphasize the current state without explicitly indicating future change.

Didn't meant to imply future expectations here.
Copied from `validateMultusDynamicNetworks`

Multus has string pretty the one you suggested.

So which one to keep ?

Reply:
I suggest we do a simplistic "'macvtap' has been requested, but is not supported on OpenShift".

Reply:
done